_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+
+This release includes support for another feature from YouTube: meta info. The little boxes sometimes telling you that a channel belongs to a news group, or giving you further information to specific search queries - they appear in NewPipe now as well. If you do not want to see those boxes, head over to <code><nobr><img src="{{ site.baseurl }}/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="{{ site.baseurl }}/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>Show meta info</nobr></code> and disable the display there. If you enjoy these boxes however, thank [@TobiGr](https://github.com/TobiGr) for bringing them into this release.
+
+In some kind of more personal news, we have a new domain now - [https://newpipe.net](https://newpipe.net). This makes us look more professional (we hope), but also meant the URLs needed to be updated in the app. This was done by [@TiA4f8R](https://github.com/TiA4f8R).
+## Bugs, be gone!
+We aim for the best NewPipe with every release we do. Because of that, we improve on issues in every update. This time around, we fixed an issue where the app would crash when you tried to delete a video from your playlist while the app was refreshing your subscriptions. This was discovered by [@gasull](https://github.com/gasull), reproduced by [@chilliger](https://github.com/chilliger) and fixed by [@hlloreda](https://github.com/hlloreda). Thanks to all three of them. 
+
+In another attempt to improve the ReCaptcha experience, [@Stypox](https://github.com/Stypox) fixed a situation where only a blank page was shown and is likely very exhausted with that topic at this point. We thank him massively for taking up this issue time and time again.
+
+Talking about time, sharing URLs from any service except YouTube now doesn't try to add a timestamp to the URL of the video. This broke the URL on any other service. In the future we hope to support the timestamps of every platform, for now, you get non-timed URLs that work. This was provided by [@B0pol](https://github.com/B0pol), thank you!
+
+The last big fix is brought to you by [@TobiGr](https://github.com/TobiGr), who got YouTube's music search to work once again. Search all the music that you want now (with the filters next to the search bar) and enjoy listing to Christmas songs you can not get out of your head.
+
+## Nerd talk
+
+There were quite some very technical/non-user-facing changes in this release. We decided to provide you this nice list with changes not affecting you at all, but we still want to publicly thank their creators:
+- The Invidious (alternative YouTube frontend which the extractor uses to access YouTube) URL list in the manifest was updated by [@opusforlife2](https://github.com/opusforlife2)
+- The ReCaptcha user agent doesn't include device informations anymore, by [@Stypox](https://github.com/Stypox) (to no ones suprise)
```

Comment:
This could potentially be of interest and impact someone, do you think it would be a bad idea to put it above, together with the other note about recaptchas?

Reply:
hmm I thought about it but also thought it would be too technical. I definitly can though. How does that sound to you?

In another attempt to improve the ReCaptcha experience, [@Stypox](https://github.com/Stypox) fixed a situation where only a blank page was shown. He also changed that the user agent sent to the ReCaptcha service does not include device information anymore. He is likely very exhausted with that topic at this point. We thank him massively for taking up this issue time and time again.

Reply:
What about "He also prevented device information to be leaked inside the user agent"

Reply:
sure if you prefere leaked.